kolibri/plugins/learn/assets/src/router.js

```diff
@@ -4,35 +4,56 @@ const VueRouter = require('vue-router');
 
 Vue.use(VueRouter);
 
-const router = new VueRouter({
-  history: false, // do not use the HTML5 history API
-});
-
-router.map({
-  '/explore': {
-    component: require('./vue/explore-page'),
-    name: 'explore-page',
-  },
-  '/explore/topic/:content_id': {
-    component: require('./vue/explore-page'),
-    name: 'explore-topic-page',
-  },
-  '/explore/content/:content_id': {
-    component: require('./vue/content-page'),
-    name: 'explore-content',
-  },
-  '/learn': {
-    component: require('./vue/learn-page'),
-    name: 'learn-page',
-  },
-  '/scratchpad': {
-    component: require('./vue/scratchpad'),
-    name: 'scratchpad',
-  },
-});
-
-router.redirect({
-  '/': '/explore',
-});
-
-module.exports = router;
+
+/** Wrapper around Vue Router.
+ *  Implements URL mapping to Vuex actions rather than Vue components.
+ */
+class Router {
+  /**
+   * Create a Router instance.
+   */
+  constructor() {
```

Comment:
Fine for MMVP - but if we want to replicate this pattern across the app, we should add a `.map` method to be equivalent to Vue-Router?